.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
https://prettier.io/docs/en/configuration.html#sharing-configurations
this configuration should probably be part of a new package (`@americanexpress/prettier-config`) and used in `.prettierrc.json` here
but need everything else about this PR finalized before making that change?

Reply:
If this is how it should be done, then we should do it that way. :)